src/NerdBank.GitVersioning/ReleaseManager.cs

```diff
@@ -56,6 +56,11 @@ public enum ReleasePreparationError
             /// HEAD is detached. A branch must be checked out first.
             /// </summary>
             DetachedHead,
+
+            /// <summary>
+            /// The versionIncrement setting cannot be applied to the current version
```

Comment:
nit: trailing period.

```suggestion
            /// The versionIncrement setting cannot be applied to the current version.
```